sre_maturity.md

```diff
@@ -175,27 +175,68 @@ In this phase you do ad-hoc security assessments, scans and pentesting.
 
-### Room to improve - Phase 3: The 'Run' Phase
+Should an SRE team fail at reducing toil faster than it accumulates due to service growth, such a team can find itself in a very serious predicament – spending all of its time on operations tasks with zero room to improve (e.g. automate) anything.
 
-*This is a stub. Please help by adding content. Ref:[SIGSRE-87](https://issues.redhat.com/browse/SIGSRE-87)*
+The chart below visualizes what happens to an SRE team's time commitments when their system is growing 20% quarter over quarter yet they do not address the toily part of operations (marked in red).
+Within a year the system more than doubles its size and a team that originally spent 40% of their time on operations and 60% on development work (a healthy balance) now sees that proportion skew towards 79% to 22%.
+Two more quarters of this and operations becomes 100% of time allocated, which makes such a team "SRE" in name only.
+
+![Engineering Time Commitments at 20% Quarter over Quarter Growth](images/Engineering%20Time%20Commitments%20at%2020%25%20Quarter%20over%20Quarter%20Growth.png)
+
+This is obviously a bad trajectory to be on for multiple reasons:
+- The system being managed will hit scaling issues.
+- The team's morale will quickly plummet and attrition – especially among the more experienced engineers – will increase, which will only worsen the underlying situation.
+
+No team should let matters get to such a state.
+Toil management practices should be properly set up from the start for any new SRE team and once the basic approach becomes insufficient, leadership should take notice and start reaching for more advanced tools (see the 'Walk' and 'Run' phases described below).
+
+But in case your team did not address these issues early enough and now all everyone does is operations and there's zero room for working on toil reduction, there's still one thing left – declare Toil Bankruptcy.
+Temporarily suspend SLOs, communicate to stakeholders you will, for a time, miss reliability objectives on your system and use the freed engineering resources to aggressively pursue toil reduction efforts.
+Start with anything your team identified as low effort and high reward.
+Only once you're certain your team has made enough improvements that there's no longer a risk of things spiralling out of control, can you return to your previous obligations.
+
+### Toil Management and Operational Improvements - Phase 2: The 'Walk' Phase
+
+*This is a stub. Please help by adding content. Ref: [SIGSRE-86](https://issues.redhat.com/browse/SIGSRE-86)*
+
+* Contact with upstream (whichever team owns the product you’re SRE-ing) is well-established and they have resources reserved to work on issues raised by your SRE team. Workarounds for production issues applied by SREs are always temporary, while upstream works on a proper fix.
+* Toil items are tracked and a simple scoring system is used to prioritize work.
```

Comment:
I think readers are going to want more out of this bullet. Some even initial/starting proposal for how to do this.